### ISSUE TITLE
Enum not properly translated to a go type

### ISSUE THREAD

**Issue by robvanmieghem** [Jumpscale/go-raml#37]
Currently the following raml type
```
  Invitation:
    properties:
        user: string
        role:
          type: string
          enum: [owner, member]
        created: date
```
is converted to 
```
type Invitation struct {
	Created Date     `json:"created"`
	Role    []string `json:"role"`
	User    string   `json:"user"`
}
```

**Comment by iwanbk** [comment]
@robvanmieghem 
I think we misinterpreted the spec 
```
Enumeration of possible values for this primitive type. Cannot be used with the file type. The value is an array containing string representations of possible values,
```
In the array part.

So, in above case, `Role` has `string` as type. The enum part won't be used in current version, it will be used in input validation.

**Comment by iwanbk** [comment]
@robvanmieghem fixed in #38
